assets/prefabs/recipe/IndianTea.prefab

```diff
@@ -0,0 +1,30 @@
+{
+    "ProcessDefinition": {
+        "processType": "Cooking:FoodMixing"
+    },
+    "CraftingStationRecipe": {
+        "recipeId": "Mixed food|Cooking:IndianTea",
+
+        "recipeComponents": ["1*Cooking:sugar","2*Cooking:tea",],
+        "recipeFluids": ["2*Fluid:Water""1*Fluid:Milk"],

```

Comment:
Could you change the milk from being a `recipeFluid` to being a plain `recipeComponent` instead? You could use `Cooking:CoconutMilk` or add a new plain milk item (we only have coconut milk right now)

The recipeFluids are meant to go in the workstation, like water for boiling eggs. There's no way to get milk as a fluid right now, although it is possible we could do that in the future (need to add cows!)